src/js/components/App.js

```diff
@@ -1,25 +1,32 @@
+import PropTypes from 'prop-types';
 import React from "react";
-import ReactDOM from "react-dom";
-import {BrowserRouter, Switch, Route, Redirect} from 'react-router-dom';
+import {Switch, Route} from 'react-router-dom';
 import Main from "./Main/Main.jsx";
 import Admin from "./Admin/Admin.jsx";
 import Login from "./Login/Login.jsx";
 
-function App(props) {
-    return (
-            <div>
-                <Switch>
-                    <Route path='/admin' component={Admin}/>
-                    <Route path='/login' component={Login}/>
-                    <Route path='/' component={Main}/>
-                </Switch>
-            </div>
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.props = props;
```

Comment:
No need to do this, `super(props)` handles this. Moreover, entire `constructor` is redundant if you do nothing with props in it (like passing them into state)